modules/qoa/doc_classes/AudioStreamQOA.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="AudioStreamQOA" inherits="AudioStream" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../doc/class.xsd">
+	<brief_description>
+		QOA audio stream driver.
+	</brief_description>
+	<description>
+		QOA audio stream driver. See [member data] if you want to load a QOA file at run-time.
```

Comment:
Not convinced about the current wording but for future reference it may be worth linking to the website:
```suggestion
		[url=https://qoaformat.org/]QOA[/url] audio stream driver. See [member data] if you want to load a QOA file at run-time.
```